packages/pass-style/src/copyArray.js

```diff
@@ -7,25 +7,32 @@ const { getPrototypeOf } = Object;
 const { ownKeys } = Reflect;
 const { isArray, prototype: arrayPrototype } = Array;
 
+/**
+ * @param {unknown} candidate
+ * @param {import('./types.js').Checker} [check]
+ * @returns {boolean}
+ */
+const canBeValid = (candidate, check = undefined) =>
+  isArray(candidate) ||
+  (!!check && check(false, X`Array expected: ${candidate}`));
+
 /**
  *
  * @type {import('./internal-types.js').PassStyleHelper}
  */
 export const CopyArrayHelper = harden({
   styleName: 'copyArray',
 
-  canBeValid: (candidate, check = undefined) =>
-    isArray(candidate) ||
-    (!!check && check(false, X`Array expected: ${candidate}`)),
+  canBeValid,
 
   assertValid: (candidate, passStyleOfRecur) => {
-    CopyArrayHelper.canBeValid(candidate, assertChecker);
+    canBeValid(candidate, assertChecker);
     getPrototypeOf(candidate) === arrayPrototype ||
       assert.fail(X`Malformed array: ${candidate}`, TypeError);
     // Since we're already ensured candidate is an array, it should not be
     // possible for the following test to fail
     checkNormalProperty(candidate, 'length', false, assertChecker);
-    const len = candidate.length;
+    const len = /** @type {unknown[]} */ (candidate).length;
```

Comment:
better at the function signature

Reply:
But not appropriate here, because `candidate` really could have any type.

Reply:
I'm confused. You're saying with the cast that it's `unknown[]`. So I'm saying to just put that in the function signature.